assets/css/style.css

```diff
@@ -6,14 +6,14 @@ a {
 
 .container {
     max-width: 1760px;
-    margin: 0 auto;
+    margin: 0 auto 200px;
```

Comment:
why did you add 200px to the margin property ? It was an idea for creating a `container` class which places our content in the center of  a page